lib/logstash/outputs/tcp.rb

```diff
@@ -191,7 +181,7 @@ def load_cert_store
 
   def initialize(*args)
     super(*args)
-    setup_ssl_params!
+    # setup_ssl_params!
```

Comment:
I think we can just stop overriding the init method from the base class now. So probably just delete the entire `initialize` method.

Reply:
Added this as a GH suggestion for ease of commit if you agree :) https://github.com/logstash-plugins/logstash-output-tcp/pull/58/files#r1876854779